packages/gestalt/src/Sticky.js

```diff
@@ -20,19 +21,28 @@ type Threshold =
 
 type Props = {|
   children: React.Node,
+  zIndex?: Indexable,
   dangerouslySetZIndex?: {| __zIndex: number |},
   ...Threshold,
 |};
 
+const DEFAULT_ZINDEX = new FixedZIndex(1);
+
 export default function Sticky(props: Props): React.Node {
-  const { dangerouslySetZIndex = { __zIndex: 1 }, children } = props;
+  const { dangerouslySetZIndex, children } = props;
+  const zIndex =
+    props.zIndex ||
+    (dangerouslySetZIndex &&
```

Comment:
Can we just plan to remove `dangerouslySetZIndex` totally in this release and codemod existing uses to `FixedZIndex` or is that too ambitious?

Reply:
I'd prefer not to couple deprecating this field and the release together. A codemod here is definitely possible but I think I'd like folks (and me) to take a moment to see if we can just fix some of the existing ones. Also - writing a codemod would require more time than I have to put into this.